src/scenic/simulators/metadrive/utils.py

```diff
@@ -79,6 +79,26 @@ def metaDriveToScenicHeading(metaDriveHeading):
     return (scenicHeading + math.pi) % (2 * math.pi) - math.pi
 
 
+def scenicToMetaDriveVelocity(scenic_velocity):
+    """Converts Scenic velocity to MetaDrive velocity by rotating the vector by π/2 (90 degrees)."""
```

Comment:
This doesn't look right to me: MetaDrive's coordinate system is only shifted from Scenic's, not rotated (`scenicToMetaDrivePosition` only applies a constant offset). So you should not need to convert the velocity vector at all.